### PR TITLE
docs: refactor loggers plugin doc (2)

### PR DESCRIPTION
**Pull request by hf400159**
### Description

Fixes # (issue)

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/zh/latest/plugins/google-cloud-logging.md [hunk 1]
```diff
@@ -114,25 +118,27 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f13
...
+```
 HTTP/1.1 200 OK
 ...
 hello, world
 ```
 
-* 登录谷歌云日志服务，查看日志
-
-[Google Cloud Logging Service](https://console.cloud.google.com/logs/viewer)
+访问成功后，你可以登录[谷歌云日志服务](https://console.cloud.google.com/logs/viewer)查看相关日志。
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/google-cloud-logging.md:133]: The previous text was in English but suddenly changed to Chinese here. It is recommended to keep it unified.

File: docs/zh/latest/plugins/google-cloud-logging.md [hunk 2]
```diff
@@ -23,38 +30,34 @@ title: google-cloud-logging
 
 ## 描述
 
-`google-cloud-logging` 插件用于将 `Apache APISIX` 的请求日志发送到 [Google Cloud Logging Service](https://cloud.google.com/logging/)。
-
-该插件提供了将请求的日志数据以批处理队列的形式推送到谷歌云日志服务的功能。
-
-有关 `Apache APISIX` 的 `Batch-Processor` 的更多信息，请参考：
-[Batch-Processor](../batch-processor.md)
+`google-cloud-logging` 插件可用于将请求日志发送到 [Google Cloud Logging Service](https://cloud.google.com/logging/)。
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/google-cloud-logging.md:33]:
```suggestion
`google-cloud-logging` 插件可用于将请求日志发送到 [Google Cloud Logging Service](https://cloud.google.com/logging/) 进行分析和存储。
```

File: docs/zh/latest/plugins/loggly.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## 描述
+
+`loggly` 插件可用于将日志转发到 [SolarWinds Loggly](https://www.solarwinds.com/loggly) 进行分析和存储。
+
+当启用插件时，APISIX 会将请求上下文信息并序列化为 [Loggly Syslog](https://documentation.solarwinds.com/en/success_center/loggly/content/admin/streaming-syslog-without-using-files.htm?cshid=loggly_streaming-syslog-without-using-files) 的数据格式，即具有 [RFC5424](https://datatracker.ietf.org/doc/html/rfc5424) 兼容标头的 Syslog。
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/loggly.md:34]:
```suggestion
当启用插件时，APISIX 会请求上下文信息，并将其序列化为 [Loggly Syslog](https://documentation.solarwinds.com/en/success_center/loggly/content/admin/streaming-syslog-without-using-files.htm?cshid=loggly_streaming-syslog-without-using-files) 的数据格式，即具有 [RFC5424](https://datatracker.ietf.org/doc/html/rfc5424) 兼容标头的 Syslog。
```

File: docs/zh/latest/plugins/loggly.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+| include_resp_body      | boolean       | 否      | false   | 当设置为 `true` 时，包含响应体。                                            |
+| include_resp_body_expr | array         | 否      |         | 当 `include_resp_body` 属性设置为 `true` 时进行过滤响应体，并且只有当此处设置的表达式计算结果为 `true` 时，才会记录响应体。更多信息，请参考 [lua-resty-expr](https://github.com/api7/lua-resty-expr)。 |
+
+该插件支持使用批处理器来聚合并批量处理条目（日志或数据）。这样可以避免插件频繁地提交数据，默认设置情况下批处理器会每 `5` 秒钟或队列中的数据达到 `1000` 条时提交数据，如需了解批处理器相关参数设置，请参考 [Batch-Processor](../batch-processor.md#配置)。
+
+如果要生成用户令牌，请在 Loggly 系统中的 `<your assigned subdomain>/loggly.com/tokens` 设置，或者在系统中单击 `Logs > Source setup > Customer tokens`。
+
+## 插件元数据设置
+
+您还可以通过插件元数据配置插件。详细配置如下：
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/loggly.md:54]:
```suggestion
你还可以通过插件元数据配置插件。详细配置如下：
```

File: docs/zh/latest/plugins/splunk-hec-logging.md [hunk 5]
```diff
@@ -23,32 +30,32 @@ title: splunk-hec-logging
 
 ## 描述
 
-`splunk-hec-logging` 插件用于将 `Apache APISIX` 的请求日志转发到 `Splunk HTTP 事件收集器（HEC）` 中进行分析和存储，启用该插件后 `Apache APISIX` 将在 `Log Phase` 获取请求上下文信息并序列化为 [Splunk Event Data 格式](https://docs.splunk.com/Documentation/Splunk/latest/Data/FormateventsforHTTPEventCollector#Event_metadata) 后提交到批处理队列中，当触发批处理队列每批次最大处理容量或刷新缓冲区的最大时间时会将队列中的数据提交到 `Splunk HEC` 中。
+`splunk-hec-logging` 插件可用于将请求日志转发到 Splunk HTTP 事件收集器（HEC）中进行分析和存储。
 
-有关 `Apache APISIX` 的 `Batch-Processor` 的更多信息，请参考：
-[Batch-Processor](../batch-processor.md)
+启用该插件后 APISIX 将在 `Log Phase` 获取请求上下文信息并序列化为 [Splunk Event Data 格式](https://docs.splunk.com/Documentation/Splunk/latest/Data/FormateventsforHTTPEventCollector#Event_metadata) 后提交到批处理队列中，当触发批处理队列每批次最大处理容量或刷新缓冲区的最大时间时会将队 列中的数据提交到 `Splunk HEC` 中。
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/splunk-hec-logging.md:35]:
```suggestion
启用该插件后，APISIX 将在 `Log Phase` 获取请求上下文信息，并将其序列化为 [Splunk Event Data 格式](https://docs.splunk.com/Documentation/Splunk/latest/Data/FormateventsforHTTPEventCollector#Event_metadata) 后提交到批处理队列中，当触发批处理队列每批次最大处理容量或刷新缓冲区的最大时间时会将队列中的数据提交到 `Splunk HEC` 中。
```

File: docs/zh/latest/plugins/splunk-hec-logging.md [hunk 6]
```diff
@@ -23,32 +30,32 @@ title: splunk-hec-logging
...
+| endpoint.token      | 是     |        | Splunk HEC 身份令牌。                                                                                                                                                |
+| endpoint.channel    | 否     |        | Splunk HEC 发送渠道标识，更多信息请参考 [About HTTP Event Collector Indexer Acknowledgment](https://docs.splunk.com/Documentation/Splunk/8.2.3/Data/AboutHECIDXAck)。 |
+| endpoint.timeout    | 否     | 10     | Splunk HEC 数据提交超时时间（以秒为单位）。                                                                                                                             |
+| ssl_verify          | 否     | true   | 当设置为 `true` 时，启用 `SSL` 验证。                                                                                                                                 |
 
-本插件支持使用批处理器来聚合并批量处理条目（日志/数据）。这样可以避免插件频繁地提交数据，默认设置情况下批处理器会每 `5` 秒钟或队列中的数据达到 `1000` 条时提交数据，如需了解或自定义批处理器相关参数设置，请参考 [Batch-Processor](../batch-processor.md#配置) 配置部分。
+本插件支持使用批处理器来聚合并批量处理条目（日志和数据）。这样可以避免该插件频繁地提交数据。默认情况下每 5 秒钟或队列中的数据达到 1000 条时，批处理器会自动提交数据，如需了解更多信息或自定义配置，请参考 [Batch-Processor](../batch-processor.md#配置)。
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/splunk-hec-logging.md:48]:
```suggestion
本插件支持使用批处理器来聚合并批量处理条目（日志和数据）。这样可以避免该插件频繁地提交数据。默认情况下每 `5` 秒钟或队列中的数据达到 `1000` 条时，批处理器会自动提交数据，如需了解更多信息或自定义配置，请参考 [Batch-Processor](../batch-processor.md#配置)。
```

**Review by SylviaBABY — DISMISSED** [review]

**Review by SylviaBABY — APPROVED** [review]

**Review by bzp2010 — APPROVED** [review]

**Pull request merged**